.ci/atime/tests.R

```diff
@@ -135,9 +135,9 @@ test.list <- atime::atime_test_list(
     expr = {
       expr=data.table:::`[.data.table`(d, , (max(v1)-min(v2)), by = id3)
     },
-    Before="793f8545c363d222de18ac892bc7abb80154e724", # This is the link to the Before commit (https://github.com/Rdatatable/data.table/commit/793f8545c363d222de18ac892bc7abb80154e724) as stated here (https://github.com/Rdatatable/data.table/issues/4200#issuecomment-646111420) 
+    Before="793f8545c363d222de18ac892bc7abb80154e724", # Commit (https://github.com/Rdatatable/data.table/commit/793f8545c363d222de18ac892bc7abb80154e724) which was Before the regression was introduced as stated in the issue (https://github.com/Rdatatable/data.table/issues/4200#issuecomment-646111420) 
```

Comment:
this is still not an acceptable comment, because "do not include the link to the parent commit https://github.com/Rdatatable/data.table/commit/cacdc92df71b777369a217b6c902c687cf35a70d as that does not help understanding how the commit is related to the PR/issue, and it is redundant with the code which already has that ID."
Please read all of https://github.com/Rdatatable/data.table/wiki/Performance-testing#documentation-of-historical-commits and write a comment which follows those rules, with minimal deviation from the templates.

Reply:
I revised that guideline to clarify:

do not include the link to the same commit ID as in the code, as that does not help understanding how the commit is related to the PR/issue, and it is redundant with the code which already has that ID. For example the code above has `Slow = "cacdc92df71b777369a217b6c902c687cf35a70d"` so it is not necessary to inclde the link https://github.com/Rdatatable/data.table/commit/cacdc92df71b777369a217b6c902c687cf35a70d

Reply:
I have updated the Before commit id and given an appropriate comment , please check